golden-container-images/rust-glibc/Dockerfile

```diff
@@ -1,6 +1,6 @@
-FROM cgr.dev/zama.ai/glibc-dynamic:15.2.0-dev AS builder
+FROM cgr.dev/zama.ai/glibc-dynamic:15.2.0-dev AS prod
```

Comment:
would it not be easier to drop our own golden image in favor of chainguard rust image?